README.rst

```diff
@@ -92,7 +92,13 @@ config option (or ``default``) will be used to namespace the relevant things:
 Configuration
 =============
 
-Run the following command to create the necessary tables in the database (ensuring the pyenv is activated)::
+Run the following command to create the necessary tables in the database (ensuring the pyenv is activated):
+
+ON CKAN == 2.9::
```

Comment:
Let's future-proof this!

```suggestion
ON CKAN >= 2.9::
```

Reply:
Same on all other `CKAN == 2.9` occurrences

Reply:
Thanks @amercader . Makes sense. Please see 6ab5ca4